lib/components/artist/header.js

```diff
@@ -57,16 +57,31 @@ class Header extends React.Component {
         <Headline style={[styles.base, styles.headline]}>
           {artist.name}
         </Headline>
-        <SerifText style={styles.base}>
-          {artist.nationality}, b. {artist.birthday}
-        </SerifText>
+        {this.renderSection(artist.nationality, artist.birthday)}
         <SerifText style={[styles.base, styles.followCount]}>
           {artist.counts.follows} Followers
         </SerifText>
         <InvertedButton text="Follow" selected={this.state.following} onPress={this.handleFollowChange} />
       </View>
     );
   }
+  
+  renderSection(nationality, birthday) {
+    const bylineRequired = (nationality || birthday);
+    const nationalityString = nationality + (birthday ? ", " : "");
+    const birthdayString = "b. " + birthday;
+    if (bylineRequired) {
+      return ( 
+        <View>
+            <SerifText style={styles.base}>
+                {nationality ? nationalityString : null}{birthday ? birthdayString : null}
+              </SerifText>
+        </View>
+      );
+    } else {
+      return (null);
```

Comment:
The parentheses are only needed around the JSX syntax and only when it spans multiple lines.